plot_continuous_monitor.py

```diff
@@ -191,6 +192,16 @@ def parse_args():
     help="Whether to open the resulting graph PDFs.")
   return parser.parse_args()
 
+def get_util_for_disk(disk_utils, disk):
+  """
+  Returns the disk utilization metrics for the specified disk, given the utilization information
+  for all disks, or None if the desired disk cannot be found.
+  """
+  for disk_util in disk_utils:
+    if disk in disk_util:
+      return disk_util[disk]
+  return None
```

Comment:
Is the None ever checked for? would it make sense to return -1 or something else that would be obviously wrong when the graph gets generated?